### PR TITLE
[RB] - Add currency and currencyISO to invoice table

### PR DESCRIPTION
**Pull request by rBangay**
## What does this change?

Add currency and currencyISO properties to the defualt returned object in the getPlan function. This targets the gift subscriptions and allows the correct data to be displayed in the invoice table on the billing page

## Images

Before             |  After
:-------------------------:|:-------------------------:
  |

### CODE REVIEW

File: app/client/components/billing/billing.tsx [hunk 1]
```diff
@@ -119,18 +118,20 @@ const BillingRenderer = ([mdaResponse, invoiceResponse]: [
                   const paidPlan = getMainPlan(
                     productDetail.subscription
                   ) as PaidSubscriptionPlan;
+                  console.log("paidPlan = ");
+                  console.log(paidPlan);
```
Comment (kelvin-chappell) [inline app/client/components/billing/billing.tsx:122]: Do you want to leave these here?

**Review by kelvin-chappell — APPROVED** [review]
Nice solution! 👍

File: app/client/components/billing/billing.tsx (hunk 1 above)
Comment (rBangay) [reply app/client/components/billing/billing.tsx:122]: ooo no :s

**Review by kelvin-chappell — APPROVED** [review]
👍

**Pull request merged**